### PR TITLE
added total /network/total-staked api route

### PR DESCRIPTION
**Pull request by miiu96**
Added /`network/total-staked` api route
This api route returns the total staked value from the validators system smart contract

### CODE REVIEW

File: api/groups/baseNetworkGroup_test.go [hunk 1]
```diff
@@ -169,6 +169,62 @@ func TestGetNetworkConfigData_OkRequestShouldWork(t *testing.T) {
 	assert.Equal(t, value, res)
 }
 
+func TestGeTotalStaked_OkRequestShouldWork(t *testing.T) {
```
Comment (andreibancioiu) [inline api/groups/baseNetworkGroup_test.go:172]: `get` typo.

File: api/groups/baseNetworkGroup_test.go [hunk 2]
```diff
@@ -169,6 +169,62 @@ func TestGetNetworkConfigData_OkRequestShouldWork(t *testing.T) {
...
+
+	var result metricsResponse
+	loadResponse(resp.Body, &result)
+
+	res, ok := result.Data[key]
+	assert.True(t, ok)
+	assert.Equal(t, value, res)
+}
+
+func TestGetTotalStaked_FacadeErrShouldErr(t *testing.T) {
```
Comment (andreibancioiu) [inline api/groups/baseNetworkGroup_test.go:204]: Do we still need this type of tests (`facade`-related), or not needed anymore? @bogdan-rosianu?

File: api/groups/baseNetworkGroup_test.go (hunk 1 above)
Comment (sasurobert) [reply api/groups/baseNetworkGroup_test.go:172]: +1

**Review by sasurobert — COMMENTED** [review]
only the typo

File: README.md [hunk 3]
```diff
@@ -47,6 +47,7 @@ For more details, go to [docs.elrond.com](https://docs.elrond.com/sdk-and-tools/
 - `/v1.0/network/status/:shard`    (GET) --> returns the status metrics from an observer in the given shard
 - `/v1.0/network/config`           (GET) --> returns the configuration of the network from any observer
 - `/v1.0/network/economics`        (GET) --> returns the economics data metric from the last epoch
+- `/v1.0/network/total-staked`     (GET) --> returns the total staked value from the validators contract
```
Comment (bogdan-rosianu) [inline README.md:50]: :+1:

File: process/nodeStatusProcessor.go [hunk 4]
```diff
@@ -63,6 +66,29 @@ func (nsp *NodeStatusProcessor) GetNetworkStatusMetrics(shardID uint32) (*data.G
 	return nil, ErrSendingRequest
 }
 
+func (nsp *NodeStatusProcessor) GetTotalStaked() (*data.GenericAPIResponse, error) {
```
Comment (bogdan-rosianu) [inline process/nodeStatusProcessor.go:69]: missing comment

File: process/nodeStatusProcessor_test.go [hunk 5]
```diff
@@ -73,6 +74,58 @@ func TestNodeStatusProcessor_GetConfigMetrics(t *testing.T) {
 
 }
 
+func TestNodeStatusProcessor_GetTotalStakedErr(t *testing.T) {
```
Comment (bogdan-rosianu) [inline process/nodeStatusProcessor_test.go:77]: add this endpoint also in the `testing` package

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: process/nodeStatusProcessor.go (hunk 4 above)
Comment (miiu96) [reply process/nodeStatusProcessor.go:69]: done.

File: process/nodeStatusProcessor_test.go (hunk 5 above)
Comment (miiu96) [reply process/nodeStatusProcessor_test.go:77]: done.

File: api/groups/baseNetworkGroup_test.go (hunk 1 above)
Comment (miiu96) [reply api/groups/baseNetworkGroup_test.go:172]: done.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**